h/auth/policy.py

```diff
@@ -12,10 +10,13 @@
 
 from h.auth import util
 
-# As we roll out the new API Auth Policy with Auth Token Policy, we
-# want to keep it restricted to certain endpoints
-# Currently restricted to `POST /api/groups*` only
-AUTH_TOKEN_PATH_PATTERN = '^\/api\/groups'
+#: List of route name-method combinations that should
+#: allow AuthClient authentication
+AUTH_CLIENT_API_WHITELIST = [
```

Comment:
I hope this is both easier to understand as a human and less error-prone than the earlier approach.